mobius/src/geometry/circular_arc.rs

```diff
@@ -15,6 +35,59 @@ impl CircularArc {
     pub fn new(circle: Circle, angles: ArcAngles) -> Self {
         Self { circle, angles }
     }
+
+    /// Given a circle and 3 points (a, b, c) on that circle, compute a circular
+    /// arc from a -> b -> c
+    pub fn from_circle_and_points(
+        circle: Circle,
+        a: Complex,
+        b: Complex,
+        c: Complex,
+    ) -> Result<Self, ArcAnglesParseError> {
+        // Determine if the 3 points circulate counterclockwise or
+        // clockwise by forming a triangle ABC and computing
+        // (the magnitude of) the wedge product.
+        let ab = b - a;
+        let ac = c - a;
+        let ccw = Complex::wedge(ab, ac) > 0.0;
+
+        // Get the raw angles
+        let theta_a = circle.get_angle(a).unwrap();
+        let theta_b = circle.get_angle(b).unwrap();
+        let theta_c = circle.get_angle(c).unwrap();
+
+        let angles = if ccw {
+            compute_ccw_angles(theta_a, theta_b, theta_c)
+        } else {
+            compute_cw_angles(theta_a, theta_b, theta_c)
+        }?;
```

Comment:
I think these could be replaced by calls to `ArcAngles::from_raw_angles(a, b, CounterClockwise|Clockwise)`